IaC/scripts/create-cluster-env.bash

```diff
@@ -0,0 +1,405 @@
+    fi
+    # Add DNS A record for the Istio ingress gateway.
+    aze network dns record-set a add-record -g $Ngsa_DNS_RG -z $Ngsa_Domain_Name -n $Ngsa_Name -a $INGRESS_PIP -o table ${subs_arg}
+
+    # Show the Azure nameservers for your DNS Zone.
+    aze network dns zone show -n $Ngsa_Domain_Name -g $Ngsa_DNS_RG --query nameServers -o tsv ${subs_arg}
+
+    ## Install Cert Manager
+    cd $REPO_ROOT/IaC/AKS/cluster/manifests/cert-manager
+    CERT_MANAGER_VERSION=1.0.3
+    kubectl create namespace cert-manager
+    echo -e "\nInstalling Cert-Manager"
+    helm install cert-manager jetstack/cert-manager --namespace cert-manager --version "v${CERT_MANAGER_VERSION}" --set installCRDs=true --atomic
+    # wait for the cert manager pods to be ready
+    kubectl get pods --namespace cert-manager
+    
+    # Create a staging and production ClusterIssuer for cert-manager
+    # Use the staging ClusterIssuer for testing. Once ready, use the production resource.
+    envsubst < "clusterissuer.yaml" | kubectl apply -f -
+
+    # set application endpoint
+    export Ngsa_App_Endpoint="${Ngsa_Name}.${Ngsa_Domain_Name}"
+    cert_enabled=true
+    Ngsa_Https_App_Endpoint="https://${Ngsa_App_Endpoint}"
+fi
+
+## Deploy NGSA with Helm
+# Directly installing PROD cert issuer
+echo -e "\nDeploying NGSA with Helm"
+cd $REPO_ROOT/IaC/AKS/cluster/charts/ngsa
+# Use the helm-config.yaml file to configure the deployment
+envsubst < helm-config.example.yaml >| helm-config.yaml
+cd $REPO_ROOT/IaC/AKS/cluster/charts
+helm install ngsa-aks ngsa -f "./ngsa/helm-config.yaml" --namespace ngsa --set cert.issuer=letsencrypt-prod --set cert.enabled=${cert_enabled} --atomic
+
+# check the version endpoint
+# you may get a timeout error, if so, just retry
+http ${Ngsa_App_Endpoint}/version
+
+## Running validation
+# run the tests in a container
+echo -e 'Running basic validation (local)'
+docker run -it --rm retaildevcrew/webvalidate --server $Ngsa_Https_App_Endpoint --base-url https://raw.githubusercontent.com/retaildevcrews/ngsa/main/TestFiles/ --files baseline.json
+## Smoke Tests
+if [[ ! -z ${Ngsa_Smoke} ]];then
+    cd $REPO_ROOT/IaC/AKS/cluster/charts/
+
+    echo -e "\nRunning smoke tests"
+    kubectl create namespace ngsa-l8r
+    cp -f ./loderunner/helm-config.example.yaml ./loderunner/helm-config.yaml
+    helm install l8r loderunner -f ./loderunner/helm-config.yaml --namespace ngsa-l8r --atomic
+    
+    kubectl get pods --namespace ngsa-l8r
+    # Fluent bit log forwarding
+    echo -e "\nForwarding FluentBit log to Azure Log Analytics"
+    cd $REPO_ROOT/IaC/AKS/cluster/charts
+
+    kubectl create namespace fluentbit
+    kubectl create secret generic fluentbit-secrets --namespace fluentbit --from-literal=WorkspaceId=${Ngsa_Log_Analytics_Wid} --from-literal=SharedKey=${Ngsa_Log_Analytics_SharedKey}
+
+    helm install fluentbit fluentbit --namespace fluentbit --atomic
+
+    # Verify the fluentbit pod is running
+    kubectl get pod --namespace fluentbit
```

Comment:
I don't think fluentbit should be dependent on the smoker install. Maybe separate it into it's own conditional block or always install fluentbit after the app setup?